### ISSUE TITLE
Proton-caller silently exits with no visible error on one machine

### ISSUE THREAD

**Issue by zerkerX** [caverym/proton-caller#24]
I have two machines at home, both running Debian Testing (11 at present). proton-call works fine with one of them (using a Radeon RX 5700), but the other machine running a GeForce RTX 1070 does not appear to function at all. I suspect the problem is something in my user configuration, but I cannot isolate it, and I am getting very little output to diagnose.

When I attempt to run a game, proton displays initial information then quits back to the terminal without running the game:

```console
$ proton-call -p 5.13 -r MSFC.exe 

________Proton________
esync: up and running.
______________________

```

I have tried using environment variables like ``DXVK_LOG_LEVEL=debug``, ``PROTON_LOG=1`` and ``PROTON_CRASH_REPORT_DIR=/home/zerker``, but I do not get any more logs. Setting ``PROTON_NO_ESYNC=true`` at least changes the startup message, but the game still does not run. I have no trouble running the same game with dxvk manually installed into a wine prefix, or with various other games running in Proton through Steam itself.

I have tried a few other games with proton-call on the same machine, with the same results. I have also manually set a drive mapping as a symlink for the folder root into my pfx/dosdevices folder, though theoretically the default z: mapping for the file system root should cover this. All partitions on my machine are ext4, if that matters.

Here are logs of the output of ``hwinfo`` and ``apt list --installed`` if that helps. If I could get any output at all to diagnoise, it would be great. Let me know if you know anything I could try.

[hwinfo.log](https://github.com/caverym/proton-caller/files/6797505/hwinfo.log)
[packages.log](https://github.com/caverym/proton-caller/files/6797506/packages.log)

**Comment by caverym** [comment]
The problem could very well be the difference in system configuration. I'd like to see your config file.
 
Also try the `--log` flag, this passes the PROTON_LOG variable when executing proton. 

My system has two GPUs, but I only ever run games on a Nvidia one; generally once in the proton block, all proton-caller functions are suspended, waiting for the proton process to exit.

**Comment by zerkerX** [comment]
Actually, in the process of trying a couple more things, I both solved it and isolated how I got in that situation. The issue was creating the prefix folder with version 6.3 then downgrading it to 5.13. If I create a fresh 5.13 prefix folder it works. I thought I had tried that, but apparently not.

I'll leave it to you to decide whether this is still a bug to be solved, or just something to throw in the FAQ.

**Comment by caverym** [comment]
I've been thinking about having PC create separate directories for every program or just each version of Proton used. It may fix this bug

**Comment by zerkerX** [comment]
That sounds like a good idea. My personal preference is probably one per version.

**Comment by caverym** [comment]
Implementing one per Proton version — so far so good

```rust
[src/lib.rs:61] self = Proton {
    steam: "/home/avery/.steam/steam/",
    proton: Steam {
        version: "6.3",
        name: "Proton 6.3",
        path: "/home/avery/.steam/steam/steamapps/common//Proton 6.3/proton",
    },
    executable: "Cargo.lock",
    passed_args: [],
    data: "/home/avery/Documents/Proton/env/",
    log: false,
}
[src/lib.rs:62] self.proton.name() = Some(
    "Proton 6.3",
)
```

**Comment by caverym** [comment]
This is now implemented in 3.0.0
